sage/config.py

```diff
@@ -11,6 +11,9 @@
 
 from sage.reranker import RerankerProvider
 
+GEMINI_MAX_CHUNKS_PER_BATCH = 64
+GEMINI_MAX_TOKENS_PER_CHUNK = 2048
```

Comment:
According to the documentation I linked above, there's also a 20k token limit per request.
You can follow the logic I have for OpenAI (where I defined `OPENAI_MAX_TOKENS_PER_JOB = 3_000_000`.

Reply:
Yea that documentation is for the vertex api which this is not. Confusing I know but I tested some of those limits and it seems they don't apply to this api